application-analytics-ui/src/main/resources/Analytics/Code/Macros/VelocityMacros.xml

```diff
@@ -190,6 +190,25 @@
   #set ($macroId = $macro + '_' + $analyticsId)
 #end
 
+#macro(displayAnalyticsMacroTitle, $macroReferencer, $descriptionTranslationKey, $nameTranslationKey)
+  #set ($discard = $xwiki.ssx.use('Analytics.Code.Macros.Description&amp;amp;Links'))
```

Comment:
This page has some JS that has to be loaded on each macro. This seems to be something that might be needed in the future, not only for description and links. So best is to just have a `Analytics.Code.AnalyticsJS` page that is loaded on each macro